### PR TITLE
Feature/stdin

### PR DESCRIPTION
**Pull request by Joinhack**
1. add stdin inherit feature.
2. support stdio command line and configure file.

### CODE REVIEW

File: bls-runtime/src/cli_clap.rs [hunk 1]
```diff
@@ -37,6 +41,12 @@ const PERMISSION_HELP: &str = "the permissions for app";
 
 const MODULES_HELP: &str = "the modules used by app";
 
+const STDOUT_HELP: &str = "the stdout setting by app, the value is inherit/null/the name of file";
+
+const STDERR_HELP: &str = "the stderr setting by app, the value is inherit/null/the name of file";
+
+const STDIN_HELP: &str = "the stdin setting by app, is inherit/null/string for input";
```
Comment (zees-dev) [inline bls-runtime/src/cli_clap.rs:48]: nit: could improve working here; hard to understand what this means

File: bls-runtime/src/config.rs [hunk 2]
```diff
@@ -189,10 +189,21 @@ impl CliConfig {
...
+        let stdin = match stdin {
+            Some(s) => {
+                if s == "inherit" {
+                    Stdin::Inherit
+                } else {
+                    Stdin::Fix(s.to_string())
+                }
+            }
+            _ => Stdin::Fix(String::new()),
+        };
```
Comment (zees-dev) [inline bls-runtime/src/config.rs:201]: nit: could implement the `TryFrom<String>` trait for `Stdin` type

File: crates/wasi-common/src/blockless/config.rs [hunk 3]
```diff
@@ -31,7 +31,13 @@ impl From<&str> for LoggerLevel {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
+pub enum Stdin {
+    Inherit,
+    Fix(String),
```
Comment (zees-dev) [inline crates/wasi-common/src/blockless/config.rs:37]: nit:
```suggestion
    Fixed(String),
```

**Review by zees-dev — APPROVED** [review]
Minor nits; LGTM 👍

File: bls-runtime/src/cli_clap.rs (hunk 1 above)
Comment (Joinhack) [reply bls-runtime/src/cli_clap.rs:48]: yes. I think so.

**Review by zees-dev — APPROVED** [review]

**Pull request merged**